target/linux/lantiq/image/ar9.mk

```diff
@@ -29,6 +29,21 @@ define Device/avm_fritz7320
 endef
 TARGET_DEVICES += avm_fritz7320
 
+define Device/avm_fritz7330
+  $(Device/AVM_preloader)
+  DEVICE_MODEL := FRITZ!Box 7330
+  SOC := ar9
+  IMAGE_SIZE := 15744k
+  LOADER_FLASH_OFFS := 0x31000
+  DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-basic-mbedtls \
+	kmod-ltq-adsl-ar9-mei kmod-ltq-adsl-ar9 \
+	kmod-ltq-adsl-ar9-fw-b kmod-ltq-atm-ar9 \
+	ltq-adsl-app ppp-mod-pppoa \
+	kmod-ltq-deu-ar9 kmod-usb-dwc2 fritz-tffs -swconfig
+  SUPPORTED_DEVICES += FRITZ7330
```

Comment:
Why do you add a `SUPPORTED_DEVICES` entry? 
This is only needed when sysupgrade should also work for images named `SUPPORTED_DEVICES`.

Reply:
This comment is no longer relevant after applying the suggested changes, so I mark it as resolved.